CMake/PackageConfig.cmake

```diff
@@ -1,13 +1,22 @@
 add_library(HighFive INTERFACE)
-target_link_libraries(HighFive INTERFACE ${HDF5_LIBRARIES})
+
+# Public headers
 target_include_directories(HighFive INTERFACE
   "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>"
   "$<INSTALL_INTERFACE:include>")
 target_include_directories(HighFive SYSTEM INTERFACE ${HDF5_INCLUDE_DIRS})
+target_link_libraries(HighFive INTERFACE ${HDF5_LIBRARIES})
+target_compile_definitions(HighFive INTERFACE ${HDF5_DEFINITIONS})
+
+# MPI
+if(HIGHFIVE_PARALLEL_HDF5)
+  target_link_libraries(HighFive INTERFACE MPI::MPI_C)
```

Comment:
Use also the back-compat format ${MPI_C_LIBRARIES} and ${MPI_C_HEADER_DIR}

Reply:
Ok, but I use `MPI_C_INCLUDE_PATH` because `MPI_C_HEADER_DIR` is only available since 3.10.